src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs

```diff
@@ -80,15 +75,21 @@ public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder ro
     [Feature("OrchardCore.ContentLocalization.ContentCulturePicker")]
     public class ContentPickerStartup : StartupBase
     {
+        private readonly IShellConfiguration _shellConfiguration;
+        public ContentPickerStartup(IShellConfiguration shellConfiguration)
+        {
+            _shellConfiguration = shellConfiguration;
+        }
+
         public override void ConfigureServices(IServiceCollection services)
         {
+            services.AddLiquidFilter<SwitchCultureUrlFilter>("switch_culture_url");
```

Comment:
think you added this one to this startup (which makes way more sense), and forgot to remove it from the above startup?

Reply:
Yes, thank you. Fixed